engine/src/main/resources/application.properties

```diff
@@ -201,3 +201,8 @@ quarkus.rest-client.export-service.trust-store-type=${clowder.private-endpoints.
 
 # Quarkus caches
 quarkus.cache.caffeine.drawer-template.expire-after-write=PT5M
+
+# REST client settings for the back office proxy, BOP or MBOP. The proxy isn't
+# a Clowder application, so the parameter must be overriden in stage and
+# production.
+quarkus.rest-client.mbop.url=http://localhost:8090
```

Comment:
I think we need a similar set of properties:

https://github.com/RedHatInsights/notifications-backend/blob/38e429815ffb216c8851504832000381ff813132/engine/src/main/resources/application.properties#L132-L135

Actually we can use the very same properties or even extend the other BOP bean.

Reply:
I guess the answer is yes, but do we need the token and client IDs? In the REST client I have not implemented those bits...

Reply:
2 weeks later... sorry! 
Yes, I believe we will need the token and client IDs to talk with BOP.